news/2023/2023-09-23-osu-originals-corsace-open-2023.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "osu! originals - Corsace Open 2023"
+date: 2023-09-23 12:00:00 +0000
+---
+
+**Corsace** — the powerhouse behind years of community tournaments and events — is making its mark in our Featured Artist catalogue with **9** original tracks!
+
+![](https://assets.ppy.sh/artists/381/header.jpg)
+
+If you watched *Corsace Open 2023* over the last few months, you probably saw this coming.
+
+<div align="center">
+    <video width="50%" controls>
+        <source src="https://assets.ppy.sh/artists/381/spoiler.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+[**Corsace**](https://corsace.io/) has been an innovator in the osu! tournament scene for years, and as part of *Corsace Open 2023*, we supported the **Corsace** team (and a dozen or so musicians) in crafting new original tracks specifically for their tournament.
+
+In other words, welcome **Corsace** to our Featured Artist collection!
+
+All **9** of the [osu! originals](/wiki/osu!_originals) from *Corsace Open 2023* can be found from [**Corsace**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/381), each pre-timed for your mapping convenience.
+
+If you're not in the mapping mood quite yet, experiencing these tunes through everything below might change your mind:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/381/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### [Aethoro - Inertia Split](https://osu.ppy.sh/beatmapsets/2044069) mapped by [LMT](https://osu.ppy.sh/users/7262798)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/381/Corsace Open 2023 Penance/Aethoro - Inertia Split.mp3">
```

Comment:
will note that while these work, links are usually url-encoded (not that i'm suggesting to change these)

Reply:
We're using a new process and it hasn't failed in any browser, so should be fine.